### PR TITLE
Change walking task instructions

### PR DESCRIPTION
**Pull request by Erin-Mounts**
https://sagebionetworks.jira.com/browse/BRIDGE-637

### CODE REVIEW

File: mPowerSDK/TasksAndSteps/APHActivityManager.m [hunk 1]
```diff
@@ -219,14 +221,31 @@ - (ORKOrderedTask *)createCustomWalkingTask
...
+    [instructionStep2 setDetailText:NSLocalizedStringWithDefaultValue(@"APH_WALKING_DESCRIPTION_2_DETAIL", nil, APHLocaleBundle(), @"Put your phone in a pocket or bag and follow the audio instructions.", @"Instructions for how to proceed to active step of walking activity.")];
     
-    ORKActiveStep *activeStep = (ORKActiveStep *)orkTask.steps[5];
-    [activeStep setTitle:titleString];
-    [activeStep setSpokenInstruction:spokenInstructionString];
+    NSString  *titleFormatWalk = NSLocalizedStringWithDefaultValue(@"APH_WALKING_WALK_TEXT_INSTRUCTION", nil, APHLocaleBundle(), @"Walk back and forth for %@ seconds.", @"Written instructions for the walking step of the walking activity, to be filled in with the number of seconds to walk.");
```
Comment (syoung-smallwisdom) [inline mPowerSDK/TasksAndSteps/APHActivityManager.m:228]: Will this translate correctly? May need to edit (but that can happen later).

File: mPowerSDK/TasksAndSteps/APHActivityManager.m [hunk 2]
```diff
@@ -219,14 +221,31 @@ - (ORKOrderedTask *)createCustomWalkingTask
...
+    
+    ORKWalkingTaskStep *walkStep = (ORKWalkingTaskStep *)orkTask.steps[3];
+    [walkStep setTitle:titleStringWalk];
+    [walkStep setSpokenInstruction:spokenInstructionStringWalk];
+    // we want to ignore steps and just go for 30 seconds, but can't disable step counting without disabling the
+    // pedometer, so we've set way more steps than 30 seconds' worth, and here we'll to set the stepDuration
+    // fallback value manually to 30 seconds.
+    walkStep.stepDuration = kWalkDuration;
+    
+    NSString  *titleFormatStand = NSLocalizedStringWithDefaultValue(@"APH_WALKING_STAND_STILL_TEXT_INSTRUCTION", nil, APHLocaleBundle(), @"Turn around 360 degrees, then stand still, with your feet about shoulder-width apart. Rest your arms at your side and try to avoid moving for %@ seconds.", @"Written instructions for the standing-still step of the walking activity, to be filled in with the number of seconds to stand still.");
```
Comment (syoung-smallwisdom) [inline mPowerSDK/TasksAndSteps/APHActivityManager.m:241]: Um... where did this text come from? Not sure it is more understandable than "turn in a full circle". Might want to verify with @cfsuver

**Comment by syoung-smallwisdom** [comment]
Few questions about the language. Otherwise, looks good.

File: mPowerSDK/TasksAndSteps/APHActivityManager.m (hunk 1 above)
Comment (Erin-Mounts) [inline mPowerSDK/TasksAndSteps/APHActivityManager.m:228]: Can't think of a language where it wouldn't, in fact AFAIK this is the recommended way to localize strings with numbers in, but yeah, can edit later if this turns out not to be the case.

File: mPowerSDK/TasksAndSteps/APHActivityManager.m (hunk 2 above)
Comment (Erin-Mounts) [inline mPowerSDK/TasksAndSteps/APHActivityManager.m:241]: From @trister (see his comment on https://sagebionetworks.jira.com/browse/BRIDGE-637), who I believe got it directly from the corresponding task in the Harvard FPHS app (which we are moving this task toward).

File: mPowerSDK/TasksAndSteps/APHActivityManager.m (hunk 1 above)
Comment (syoung-smallwisdom) [inline mPowerSDK/TasksAndSteps/APHActivityManager.m:228]: Oh, I meant the `%@ seconds`. I've seen this localized to include the seconds in a localized time interval.  That said, I decided against that for the Medication Tracking b/c it was a bit overkill. ;)

File: mPowerSDK/TasksAndSteps/APHActivityManager.m (hunk 2 above)
Comment (syoung-smallwisdom) [inline mPowerSDK/TasksAndSteps/APHActivityManager.m:241]: Okay, cool. Just seems a bit obtuse. Thanks.

**Comment by syoung-smallwisdom** [comment]
Looks good.

File: mPowerSDK/TasksAndSteps/APHActivityManager.m (hunk 1 above)
Comment (Erin-Mounts) [inline mPowerSDK/TasksAndSteps/APHActivityManager.m:228]: Oh, yeah. For handling singular vs. plural vs. other-weird-number-groupings-English-doesn't-have-but-some-languages-do on the "seconds" part.

I'd say let's go with the "don't solve the problem until we need to" philosophy here...

File: mPowerSDK/TasksAndSteps/APHActivityManager.m (hunk 2 above)
Comment (Erin-Mounts) [inline mPowerSDK/TasksAndSteps/APHActivityManager.m:241]: agreed, but there you go.

File: mPowerSDK/TasksAndSteps/APHActivityManager.m (hunk 1 above)
Comment (syoung-smallwisdom) [inline mPowerSDK/TasksAndSteps/APHActivityManager.m:228]: Yup.

**Pull request merged**